adapters/concurrent-ruby/example/concurrent-ruby.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rubygems'
+require 'bundler/setup'
+
+Bundler.require
+
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Adapters::ConcurrentRuby'
+end
+
+tracer = OpenTelemetry::Adapters::ConcurrentRuby::Adapter.instance.tracer
```

Comment:
This might be present in the other PRs, but really, you should be getting a named tracer from the OpenTelemetry.tracer_provider for tracing application code. Something like:

```
OpenTelemetry.tracer_provider.tracer('default')
```

What you have works, but we should set a good example in our examples 🙂 .